doc/api/stream.md

```diff
@@ -1596,6 +1596,7 @@ const cleanup = finished(rs, (err) => {
 ```
 
 ### `stream.pipeline(source[, ...transforms], destination, callback)`
+### `stream.pipeline(streams, callback)`
```

Comment:
This way of referencing multiple signatures is just awful, fwiw. Eventually, we really do need to go back and completely rearchitect our doc structure.